common/src/main/java/org/apache/falcon/metadata/GraphUtils.java

```diff
@@ -81,4 +87,51 @@ public static String edgeString(final Edge edge) {
                 + edge.getVertex(Direction.IN).getProperty("name")
                 + "]";
     }
+
+    public static Vertex findVertex(Graph graph, String name, String type) {
+        LOG.debug("Finding vertex for: name={}, type={}", name, type);
+        RelationshipType relationShipType;
+        switch (type) {
+        case "FEED":
+            relationShipType = RelationshipType.FEED_ENTITY;
+            break;
+        case "PROCESS":
+            relationShipType = RelationshipType.PROCESS_ENTITY;
+            break;
+        default:
```

Comment:
It appears that GraphUtils.findVertex() is a public utility method. It's legitimate to get relationshipTypes other than FEED and PROCESS as defined in the RelationshipType.java. if this method is used by this feature, it needs to be renamed or moved to some place else.